plugins/stack-overflow/src/icons/StackOverflowIcon.tsx

```diff
@@ -28,10 +30,10 @@ export const StackOverflowIcon = () => {
       fill="none"
       xmlns="http://www.w3.org/2000/svg"
     >
-      <path d="M84.021 75.825v31H11v-31H0v42h95v-42H84.021z" fill="#BCBBBB" />
+      <path d="M84.021 75.825v31H11v-31H0v42h95v-42H84.021z" fill={grey[400]} />
       <path
         d="M21.057 96.825H74v-10H21.057v10zm1.058-23.915l51.428 10.794 2.117-10.265L24.23 62.645 22.115 72.91zm6.773-24.656l47.618 22.222 4.445-9.524L33.33 38.73l-4.444 9.524zm13.227-23.386l40.423 33.65 6.773-8.042-40.53-33.65-6.666 8.042zM68.147 0L59.68 6.243l31.323 42.222 8.465-6.243L68.147 0z"
-        fill="#F48024"
```

Comment:
I am not exactly sure if we should change the logo colors unless there is a design definition that all icons should be the same color or follow the Material UI theme palette... I will speak with our team about it

Reply:
> Spoke with team today and indeed this PR doesn't provide that much value, so we're going to close this PR and open a new one with only color changes that can be made theme aware, just like you mentioned:
> 
> * Black & white colors for `theme.palette.common.white` & `theme.palette.common.black` which some themes may change to not use pure white and black as I've discussed about before.
> * Specifically searching for any colors that look remotely close to those on `theme.palette.status.*`
> 
> Should I just close this PR and open another or just revert all changes before proceeding?

We also prefer your suggestion of only making color changes that can be made based on the theme in a new pull request. Thanks again for contributing 🎉 , looking forward to seeing your new pull request.